.github/workflows/test_everest.yml

```diff
@@ -78,6 +78,16 @@ jobs:
         uv pip install git+https://github.com/equinor/everest-models.git
         ERT_PYTEST_ARGS='--cov=ert --cov=everest --cov=_ert --cov-report=xml:cov1.xml --junit-xml=junit.xml -o junit_family=legacy -m everest_models_test' uv run just everest-tests
 
+    - name: Run tests requiring everviz
+      if: inputs.test-type == 'everviz-test'
+      run: |
+        uv pip install dash==2.17.1 # https://github.com/equinor/komodo-releases/issues/6431
```

Comment:
Since ert is an open repo, and komodo-releases is a closed repo, the reason for pinning should probably be excplicitly stated here.